gpu/README.md

```diff
@@ -174,26 +174,49 @@ is specified in [third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl](https:
    to test executables. The flag is false by default to avoid unwanted coupling
    of Google-released Python wheels to CUDA binaries.
 
-5. To enforce CUDA forward compatibility mode, add
-   `--@cuda_driver//:enable_forward_compatibility=true` flag to your bazel
-   command. You can provide it either directly in a shell or in `.bazelrc`:
+5. The NVIDIA driver contains both the user mode CUDA driver and kernel mode
```

Comment:
Since  you are using UMD term afterward without explaining what it is please introuduce them  here it here: 

"user mode  driver (UMD) and kernel mode driver (KMD)", instead of just "user mode CUDA driver and kernel mode"